pkg/scheduler/plugins/multihierarchyelasticquota/quota_operator.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package multihierarchyelasticquota
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/koordinator-sh/koordinator/apis/scheduling/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	QuotaKoordinatorPrefix                         = "koordinator.quota"
+	LabelQuotaIsParent                             = QuotaKoordinatorPrefix + "/is-parent"
+	LabelQuotaParent                               = QuotaKoordinatorPrefix + "/parent"
+	LabelAllowLentResource                         = QuotaKoordinatorPrefix + "/allow-lent-resource"
+	AnnotationSharedRatio                          = QuotaKoordinatorPrefix + "/shared-ratio"
+	SystemQuotaName                  QuotaNameType = "system"
+	NeedScaleMinQuotaWhenOverRootRes               = QuotaKoordinatorPrefix + "/need-scale-minquota-when-over-rootres"
```

Comment:
Hey, what does "need-scale-minquota-when-over-rootres" mean. In the quota proposal, there is no definition of "need-scale-minquota-when-over-rootres"

Reply:
need-scale-minquota-when-over-rootres is designed according to the PR https://github.com/koordinator-sh/koordinator/pull/398/ Configuration Limit 1. But after discussion with buptcozy, need-scale-minquota-when-over-rootres should be global config instead of per quota config, So "NeedScaleMinQuotaWhenOverRootRes" is deleted.